apps/nightscout/nightscout.star

```diff
@@ -0,0 +1,210 @@
+"""
+Applet: Nightscout
+Summary: Shows Nightscout CGM Data
+Description: Displays Continous Glucose Monitoring (CGM) data from the Nightscout Open Source project (https://nightscout.github.io/).
+Author: Jeremy Tavener
+"""
+load("render.star", "render")
+load("http.star", "http")
+load("time.star", "time")
+load("encoding/base64.star", "base64")
+load("encoding/json.star", "json")
+load("cache.star", "cache")
+load("schema.star", "schema")
+
+COLOR_RED = "#f00"
+COLOR_YELLOW = "#ff0"
+COLOR_GREEN = "#0f0"
+COLOR_GREY = "#666"
+COLOR_WHITE = "#fff"
+
+DEFAULT_NORMAL_HIGH = 150
+DEFAULT_NORMAL_LOW = 100
+DEFAULT_URGENT_HIGH = 200
+DEFAULT_URGENT_LOW = 70
+
+def get_schema():
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Text(
+                id = "nightscout_id",
+                name = "Nightscout ID",
+                desc = "Your Nightscout ID (i.e. XXX in https://XXX.herokuapp.com/api/v1/entries.json)",
+                icon = "user"
+            ),
+            schema.Text(
```

Comment:
Would a `schema.Number` be useful here? Would be happy to get it added if it would be! It's non-blocking for this PR though

Reply:
Yes, that would be **AWESOME**!